### ISSUE TITLE
Maybe we should shut down our scanners to avoid them being banned.

### ISSUE THREAD

**Issue by davidc1223** [PokeAlert/PokeAlert#187]
Speculation here: https://www.reddit.com/r/pokemongodev/comments/54j1y8/0350_is_now_dead_upgrade_to_continue/d82f0es

TL:DR?

Basically Niantic forced all legitimate clients to upgrade to .39. Currently, our fake accounts are requesting information via .35. This small change can seemingly give Niantic the ability to flag scanning accounts for bans, because it is almost impossible for legitimate accounts to be requesting via .35. To be safe, I am scanning using only half of my accounts so that I don't lose ALL of them should this be true, but I'm wondering what the PokeAlert dev or others may think about this. Creating accounts is tedious and I envision a day where PTC won't allow one IP to create dozens of accounts in such a short window of time, which leads me to believe that in the long run, it might be a good idea to cease scanning until the new API is reverse engineered and PokeAlert is back online. Thoughts?

**Comment by PokeAlert** [comment]
It is too late already (the switch happened monday at 11am gmt+0) might as well use them until the end.
